modules/planning/tasks/deciders/path_bounds_decider.cc

```diff
@@ -315,6 +322,38 @@ bool PathBoundsDecider::GetBoundaryFromLanesAndADC(
 
     // 2. Get the neighbor lane widths at the current point.
     double curr_neighbor_lane_width = 0.0;
+    hdmap::Id neighbor_lane_id;
+    if (lane_borrow_info == LaneBorrowInfo::LEFT_BORROW) {
+      // Borrowing left neighbor lane.
+      if (reference_line_info.GetNeighborLaneInfo(
+              ReferenceLineInfo::LaneType::LeftForward,
```

Comment:
`using ...::LaneType;` at the beginning to shorten your code, as it's used many times.

Reply:
Modified.